source/common/router/config_impl.h

```diff
@@ -258,9 +260,10 @@ class VirtualHostImpl : public VirtualHost, Logger::Loggable<Logger::Id::router>
   };
 
   struct CatchAllVirtualCluster : public VirtualClusterBase {
+    static constexpr char other_vc_name_[] = "other";
```

Comment:
I think rather than doing this we should have the name be optional.

Reply:
Changed it to an optional instead. Thanks!